### DISCUSSION TITLE
Release v1.12.7 / 5.67.7

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3556] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

### Release Notes

This is a maintenance release, it fixes many issues, for a full list of changes and fixes please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#1127--5677---2024-01-10).

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.12.7'>Release v1.12.7 / 5.67.7</a>.</em>

**Comment by tiagozip** [comment]
Happy new year btw! I just stumbled upon sandboxie+ and I'm very happy to see it's still being manteined.

**Comment by BilleBarrett** [comment]
Question: Is there an issue with Zoom and Sandboxie just upgraded a client to windows 11 and now Zoom sound and I believer picture is not working
Can I restrict Sandboxie for look at Zoom and how 
Thank you
Bill e. Barrett

**Reply by isaak654** [reply]
Please refer to issue #579
